flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/SimpleStateRequestHandler.java

```diff
@@ -597,6 +612,13 @@
             mapStateDescriptor =
                     new MapStateDescriptor<>(
                             stateName, ByteArrayWrapperSerializer.INSTANCE, valueSerializer);
+            if (stateDescriptor.hasStateTtlConfig()) {
```

Comment:
The state ttl config is transferred in each state request. Is it possible to optimize it?